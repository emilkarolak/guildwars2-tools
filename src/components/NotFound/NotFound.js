import React from 'react';
import { NavLink } from 'react-router-dom';
import './NotFound.css';

export default () => {
  return (
    <section className="page-content">
      <h1>Error 404</h1>
      <h2>Page not found.</h2>
      <p>
        <NavLink to="/" exact className="nav-link">Go back to start</NavLink>
      </p>
    </section>
  );
}