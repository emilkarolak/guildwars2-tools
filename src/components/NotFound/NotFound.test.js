import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

it('renders without crashing', () => {
  shallow(<BrowserRouter><NotFound /></BrowserRouter>);
});

it('renders correctly', () => {
  const tree = renderer
    .create(<BrowserRouter><NotFound /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});