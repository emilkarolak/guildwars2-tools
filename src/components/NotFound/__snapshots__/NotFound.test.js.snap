// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="page-content"
>
  <h1>
    Error 404
  </h1>
  <h2>
    Page not found.
  </h2>
  <p>
    <a
      aria-current="page"
      className="nav-link active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Go back to start
    </a>
  </p>
</section>
`;
